src/app/loaders/express/__tests__/helmet.spec.ts

```diff
@@ -10,11 +9,11 @@ import helmetMiddlewares from '../helmet'
 
 describe('helmetMiddlewares', () => {
   jest.mock('helmet')
-  const mockHelmet = mocked(helmet, true)
+  const mockHelmet = jest.mocked(helmet)
```

Comment:
note to reviewer:
the old `mocked` from `ts-jest/utils` has a second arg, `deep`, which specifies whether to mock the object deeply and defaults to `false`.

the new `jest.mocked` has a second arg, `options?: { shallow: boolean }` which defaults to `false`, i.e. deep mocking is enabled by default.